src/Sulu/Bundle/TagBundle/Resources/config/lists/tags.xml

```diff
@@ -40,11 +40,12 @@
             name="name"
             visibility="always"
             searchability="yes"
-            filter-type="string"
             translation="sulu_tag.tag"
         >
             <field-name>name</field-name>
             <entity-name>%sulu.model.tag.class%</entity-name>
+
+            <filter type="text" />
```

Comment:
shouldnt the user use the search bar? i guess there is only a single tab with a specific name?